.github/workflows/bump_plugins.yml

```
@@ -0,0 +1,103 @@
+name: Update plugins
+on:
+  workflow_dispatch:
+
+jobs:
+  # ugly way to share the plugin list
+  plugin_list:
+    runs-on: ubuntu-latest
+    outputs:
+      plugins: ${{ steps.set-list.outputs.plugins }}
+    steps:
+    - name: Write the plugin list
+      run: echo '["foreman_ansible"]' > plugin_list.json
+    - name: Set the plugins list
+      id: set-list
+      run: |
+        PLUGINS=$(echo $(cat plugin_list.json))
+        echo "::set-output name=plugins::$PLUGINS"
+
+  # Bump the plugins on RPM
+  bump_plugin_rpm:
+    needs: plugin_list
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        plugin:
+          - ${{ fromJson(needs.plugin_list.outputs.plugins) }}
+
+    env:
+      PLUGIN_DIR: "packages/plugins/rubygem-${{ matrix.plugin }}"
+      SPEC_FILE: "packages/plugins/rubygem-${{ matrix.plugin }}/rubygem-${{ matrix.plugin }}.spec"
+    steps:
+    - name: Install dependencies
+      run: |
+        sudo apt-get install -y rpm curl jq git-annex python3-pip
+        sudo pip3 install semver
+        sudo gem install gem2rpm
+        sudo apt-get install -y libcurl4-openssl-dev
+        sudo curl --create-dirs -o /usr/local/bin/spectool https://pagure.io/rpmdevtools/raw/26a8abc746fba9c0b32eb899b96c92841a37855a/f/spectool.in
+        sudo chmod +x /usr/local/bin/spectool
+    - name: Fetch latest plugin gem info
+      id: latest_gem
+      run: |
+        curl_output=$(curl -s https://rubygems.org/api/v1/gems/${{ matrix.plugin }}.json)
+        echo "::set-output name=version::$(jq -r .version <<<$curl_output)"
+        echo "::set-output name=uri::$(jq -r .gem_uri <<<$curl_output)"
+    - name: Checkout RPM
+      uses: actions/checkout@v2
+      with:
+        ref: rpm/develop
+    - name: Initialize git annex
+      run: git annex init
+    - name: Try to bump plugin
+      run: |
+        alias rpmdev-packager="echo $RPM_PACKER"
+        ./bump_rpm.sh $PLUGIN_DIR
+      env:
+        RPM_PACKER: Github Actions <noreply@github.com>
+        SKIP_GIT_COMMIT: 1
+    - name: Open a PR
+      uses: peter-evans/create-pull-request@v3
+      with:
+        commit-message: "Update ${{ matrix.plugin }} to ${{ steps.latest_gem.outputs.version }}"
+        branch: "bump_rpm/${{ matrix.plugin }}"
+        title: "Update ${{ matrix.plugin }} to ${{ steps.latest_gem.outputs.version }}"
+        body: ''
+        delete-branch: true
+
+  # Bump the plugins on DEB
+  bump_plugin_deb:
+    needs: plugin_list
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        plugin:
+          - ${{ fromJson(needs.plugin_list.outputs.plugins) }}
+    steps:
+    - name: Set plugin package name
+      id: package_info
+      run: echo "::set-output name=name::$(echo ${{ matrix.plugin }} | tr _ -)"
```

Looking at it further, I think this is the right place for the real package name.
```suggestion
      run: echo "::set-output name=name::ruby-$(echo ${{ matrix.plugin }} | tr _ -)"
```